### PR TITLE
Added more blocked Prefixs!

### PR DESCRIPTION
**Pull request by AlphaGaming**

### CODE REVIEW

**Comment by PacksGamingHD** [comment]
this one is actually somewhat helpful

**Comment by JeromSar** [comment]
@AlphaGaming This PR is on hold, there is no associated issue.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 {
     public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
     {
-        "admin", "owner", "moderator", "developer", "console"
+        "admin", "owner", "moderator", "developer", "console", "sa", "sra", "sta", "0wner", "admim", "server", "founder", "dev"
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: Did you mean to spell admin "admim" the second time?

Comment (AlphaGaming) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: Yes I did mean to do that because OPs make there Tag [Admim] and it looks the same

Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java]: Ahh Okay. I saw it and assumed it was a typo.

**Comment by Wild1145** [comment]
@JeromSar Re-Open?

**Pull request closed without merging**